### ISSUE TITLE
support turbopack bundler

### ISSUE THREAD

**Issue by danielroe** [nuxt/framework#8472]
https://turbo.build/pack/docs

**Comment by wesbragagt** [comment]
I would love to see that happen for Nuxt2 and beyond

**Comment by dev-adewale** [comment]
I would advice to wait till Vite starts using Turbopack under the hood, Nuxt shouldn't port to Turbopack for bundling. Keep it in the ecosystem, what do y'all think? 💭

**Comment by podlebar** [comment]
i would advice to wait and see if Turbopack really can keep what it promises.. until now it's pure marketing without proper or real benchmarks.

**Comment by dev-adewale** [comment]
> i would advice to wait and see if Turbopack really can keep what it promises.. until now it's pure marketing without proper or real benchmarks.

I agree with you on that, even Evan You perform a benchmark test on it today. When compared to Vite, Turbopack is even slower
